reference/catalogue.v1.yaml

```diff
@@ -0,0 +1,276 @@
+openapi: 3.0.0
+info:
+  title: Provider Catalogue API
+  version: '1.0'
+  description: |-
+    This API enables a learning application provider (or anyone who wants to sell products in a Marketplace) to provide information about their products to a Marketplace.
+
+    It is envisaged that the service implementing this API, also implements the generic [Events](https://stichtingsem.stoplight.io/docs/sem-technology-prototype/reference/events.v1.yaml) and [Webhooks](https://stichtingsem.stoplight.io/docs/sem-technology-prototype/reference/webhooks.v1.yaml) API to enable subscription to changes vs polling.
+
+    ## Scopes and Data
+
+    It is envisaged that when a connection is made between a Marketplace and a Learning Application, that the API Key then provided is connected to that specific marketplace. 
+
+    This connection ensures that the Learning Application can provide information that may be specific to commercial agreements with a specific marketplace (e.g. pricing information).
+
+    This specification provides the placeholders for this information to be exchanged, but has no restriction on what is agreed between parties offline.
+  contact:
+    name: Stichting SEM
+    url: 'https://stichtingsem.org/'
+servers:
+  - url: 'https://example.stichtingsem.org/api'
+    description: catalogue
+paths:
+  /products:
+    get:
+      summary: Get Products from Catalogue
+      tags: []
+      parameters:
+        - in: query
+          name: subjectCode
```

Comment:
subjectcode is different for for the levels. The code for "English" is different for vmbo-gt-1 and havo-1. some times they share a code (havo-1, vwo-1) but i would place the subject under the level in the hierarchy

Reply:
In filtering I assume you may search for just one or the other?  

`?level=havo-1`
`?subjectCode=math-subject-code&level=havo-1`

I think the question is if you create a tuple with { level, subjectCode } - do you always have to filter by both?

Reply:
I've added an issue to discuss: https://github.com/stichtingsem/technology-prototype/issues/39